### ISSUE TITLE
Sensors don't update on 1.3 releases - Work fine on 1.2.0-beta

### ISSUE THREAD

**Issue by davidcoulson** [explosivo22/rinnaicontrolr-ha#34]
Since upgrading from 1.2.0-beta to 1.3 (including 1.3.2 this morning) my sensors such as temperature and flow rate won't update for hours even though there is activity on the water heater. I used HACS to roll back to 1.2.0-beta and now i am getting accurate and periodic updates.

Any idea if this is a config issue on my end, or a bug with the new release?

**Comment by explosivo22** [comment]
Can you make sure under the configure option of the integration that the enable maintenance retrieval is enabled when on 1.3. I made it so it can be disabled as to not interrupt the screens on the physical water heater, but it should default to enabled. I double checked and all of mine have been updating. If yours do not, I will investigate further.

**Comment by davidcoulson** [comment]
I just installed 1.3.2 again and confirmed the "Enable scheduled maintenance retrieval (Every 5 min)" option is checked already. I'll monitor it for a bit and see if it still doesn't update.

**Comment by davidcoulson** [comment]
1.3.2 seems to update every 5 minutes, although the water heater 'heating' sensor updates almost immediately. Is there a tunable parameter to pull the temperature/flow rate data more frequently?

Was also curious if we can pull any error data from the unit with this integration? My system threw a code 10 the other day, which was reported in the app (although no notification). Would be great to either flag a 'problem' sensor if there is an active error code, or report back the error code itself.

**Comment by wrmacj** [comment]
I had the same problem after the upgrade.  
![image](https://user-images.githubusercontent.com/109797056/202244529-90f54401-350a-4e10-a505-9ce8d2c67395.png)
I confirmed the "Enable scheduled maintenance retrieval (Every 5 min)" option is checked.
I then unchecked the option selected submit and then went back in and re checked the  "Enable scheduled maintenance retrieval (Every 5 min)" option.
It now seems to be working.
